src/Common/Config/configReadClient.cpp

```diff
@@ -10,14 +10,25 @@ namespace DB
 {
 bool configReadClient(Poco::Util::LayeredConfiguration & config, const std::string & home_path)
 {
+    auto safe_exists = [](const auto & path)
+    {
+        std::error_code ec;
+        bool res = fs::exists(path, ec);
+        if (ec.value() > 0)
```

Comment:
```suggestion
        if (ec)
```

https://en.cppreference.com/w/cpp/error/error_code/operator_bool

Reply:
Fixed